go/pio/cmd/pin-perf/main.go

```diff
@@ -0,0 +1,200 @@
+// Copyright 2016 Marc-Antoine Ruel. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+// pin-perf does a simple performance test of gpio via sysfs vs. gpio using memory mapped regs
+package main
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/maruel/dlibox/go/pio/host"
+	"github.com/maruel/dlibox/go/pio/host/sysfs"
+	"github.com/maruel/dlibox/go/pio/protocols/gpio"
+)
+
+const (
+	// Sleep for a short delay as there can be some capacitance on the line,
+	// requiring a few CPU cycles before the input stabilizes to the new value.
```

Comment:
Remove this block, it's not used. Funny that you did not have problems, will have to revisit my test setup.